.github/workflows/release.yml

```diff
@@ -27,3 +27,116 @@ jobs:
               with:
                   upload_url: ${{ github.event.release.upload_url }}
                   asset_path: ${{ github.workspace }}/bazel-bin/spines/pi3hat_spine
+
+    bullet_ubuntu_spine:
+      name: "Add Bullet Linux spine to release assets"
+      runs-on: ubuntu-20.04
```

Comment:
This solves the glibc compatibility issue on my computer, but at any rate we know the binaries we compile ere won't be executable by some users (who will then have to compile from source). Can we make the experience seamless for these users?

The ideal would be that, after downloading, we check if the binary runs successfully, and if not issue a message and compile from source.

This is something we didn't foresee and it extends the scope of this PR. Feel free to resolve this comment and spin this point off as a follow-up issue :ok_hand:

Reply:
Testing return code now